lib/BannerPlugin.js

```diff
@@ -12,6 +12,21 @@ const Template = require("./Template");
 const validateOptions = require("schema-utils");
 const schema = require("../schemas/plugins/BannerPlugin.json");
 
+/**
+ * BannerPlugin Options class.
+ * @typedef {Object|string} BannerOptions
+ * @property {string} banner The banner as string, it will be wrapped in a comment
```

Comment:
`Function` is also valid here.

Reply:
More here: https://github.com/webpack/webpack/blob/master/lib/BannerPlugin.js#L35-L39

Reply:
Has the BannerPlugin been updated meanwhile??? I can see the lines on the server version... will add the change. Thanks!

Reply:
after `git pull upstream master` I do: `yarn lint`
1.) ESLint couldn't find the plugin "eslint-plugin-jest".
2.) Error: Cannot find module 'fast-deep-equal'

Reply:
Okay, let's try to debug this. First, be sure to have to correct upsteam:

```
% git remote -v
origin		git@github.com:Legends/webpack.git (fetch)
origin		git@github.com:Legends/webpack.git (push)
upstream	git@github.com:webpack/webpack.git (fetch)
upstream	git@github.com:webpack/webpack.git (push)
```

Then update your `upsteam` remote:

```
% git fetch upstream
```

You can now rebase your branch:

```
% git rebase upstream/master
```

You could also reset your `master` to `upstream/master` to keep your branches in sync:

```
% git checkout master
% git reset --hard upstream/master
```

Once your branch is cleaned up, running `yarn` should fix your dependency issues.

Reply:
Thanks a lot @ooflorent, no dep issue anymore, but two lint errors in BannerPlugin.js:
"webpack-sources" and "schema-utils" is not found (node/no-missing-require) line 8 + 12

Reply:
Restarting VsCode helps....

Reply:
After "Ran all test suites" I also get this red colored message, duno if it's important:

Force exiting Jest
Have you considered using `--detectOpenHandles` to detect async operations that kept running after all tests finished?